pom.xml

```diff
@@ -7,7 +7,7 @@
         <groupId>io.spring.platform</groupId>
         <artifactId>platform-bom</artifactId>
         <version>Brussels-SR5</version>
-        <relativePath/>
+        <relativePath />
```

Comment:
Why this empty relativePath?

Reply:
The default is ../pom.xml and so this avoids Maven looking in the parent dir for a local parent. Setting it empty forces Maven to look at ~/.m2 or else remote repository.